app/javascript/components/public_health/Import.js

```diff
@@ -155,6 +169,35 @@ class Import extends React.Component {
     );
   }
 
+  getValidationErrorText(listOfErrors) {
+    let formattedErrorMessages = [];
+    listOfErrors.forEach(error => {
+      error.message.forEach(message => {
+        formattedErrorMessages.push(`${_.startCase(error.prop)} ${message}.`);
+      });
+    });
+
+    let prevent = 'prevents';
+    if (formattedErrorMessages.length > 1) {
+      prevent = 'prevent';
+    }
+
+    const heading = `The following ${pluralize('error', formattedErrorMessages.length)} ${prevent} this monitoree from being imported:`;
```

Comment:
I don't really feel like it's necessary to install a new package for this, especially since you do the same kind of check above but without this package.